exchange/howto/requirements-for-adding-an-alternative-basecurrency.adoc

```diff
@@ -0,0 +1,61 @@
+= Requirements for making a request for adding an alternative base currency
```

Comment:
Please rebase away (or revert) the commit that added this doc, so that the doc is no longer included in this PR, and issue a separate PR from a separate branch based on origin/master, taking into consideration the review feedback given above. I understand that these are two docs are related to one another on some level, but I'd prefer that each doc stand on its own, get submitted and reviewed on its own, etc. That'll keep things from piling up, and it'll keep commit histories, reviews and comments that much more uniform and simple as we go.